util/git_pre-commit.in

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright (c) 2017, Juan Hernando <juan.hernando@epfl.ch>
+
+GIT=${GIT_EXECUTABLE}
+CLANG_FORMAT=${CLANG_FORMAT}
```

Comment:
And what if your git or clang format binaries are not in the path and you detected them with CMake?